xml/ha_config_basics.xml

```diff
@@ -2264,7 +2264,7 @@ monitor_0     interval=5s timeout=20s</screen>
      <term>Value is <literal>0</literal>:</term>
      <listitem>
       <para>
-       This is the default. The resource will be placed optimally in the
+       The resource will be placed optimally in the
```

Comment:
I wouldn't remove this. Since this is actually the REAL "default" in terms of cluster configuration in pacemaker.

It's just that users shouldn't consider whatever things set by bootstrap are "defaults".

Reply:
Ahh, I see. So I leave it as it is for the time being. Thanks Yan! :+1: